GitUI/CommandsDialogs/SettingsDialog/Pages/ConsoleStyleSettingsPage.Designer.cs

```diff
@@ -49,17 +53,39 @@ private void InitializeComponent()
             this.groupBoxConsoleSettings.Margin = new System.Windows.Forms.Padding(17, 2, 3, 2);
             this.groupBoxConsoleSettings.Name = "groupBoxConsoleSettings";
             this.groupBoxConsoleSettings.Padding = new System.Windows.Forms.Padding(3, 2, 3, 2);
-            this.groupBoxConsoleSettings.Size = new System.Drawing.Size(1653, 89);
+            this.groupBoxConsoleSettings.Size = new System.Drawing.Size(1653, 130);
             this.groupBoxConsoleSettings.TabIndex = 1;
             this.groupBoxConsoleSettings.TabStop = false;
             this.groupBoxConsoleSettings.Text = "Console settings (restart required)";
             // 
+            // label2
+            // 
+            this.label2.AutoSize = true;
+            this.label2.Location = new System.Drawing.Point(6, 54);
+            this.label2.Name = "label2";
+            this.label2.Size = new System.Drawing.Size(64, 15);
+            this.label2.TabIndex = 4;
+            this.label2.Text = "Font name";
+            // 
+            // cboFontName
+            // 
+            this.cboFontName.FormattingEnabled = true;
+            this.cboFontName.Items.AddRange(new object[] {
+            "Consolas",
+            "Firacode",
+            "Cascadia Code PL"});
```

Comment:
These fonts are non standard, and must be installed by a user prior to their use. Hence we can't include them like this.
In general this isn't a good way to facilitate user settings because some users may want to user another arbitrary font. It should be done similar to how we configure other fonts, refer to the Appearance>Fonts tab.

Reply:
Thanks for the review. The combo box is currently editable so the combo box isn't only limited to items in the list. Having said that I wanted to use the standard "button based approach" for font selection but ran into a problem with the designer.

![image](https://user-images.githubusercontent.com/16128850/124413040-2d9b1000-dd93-11eb-8b78-038270fe95cc.png)

![image](https://user-images.githubusercontent.com/16128850/124413585-4c4dd680-dd94-11eb-8fbe-1e3df81a5e55.png)

This means I have a really hard time adding a change like this. I've been hand editing the designer.cs thus far. I was hoping to land this PR and do it as another PR once I figured out the issues with the designer views.

Reply:
@RussKie I implemented the change using the font selection dialog.

Reply:
@RussKie I've made the required changes now and used the font dialog as requested.